### DISCUSSION TITLE
How can I get tesnorflow working?

### DISCUSSION THREAD

**Discussion by Felixkruemel** [OpenBB-finance/OpenBB#239] [Q&A] [answered]
If I want to do some prediction tensorflow won't detect my 2060Super witht he following error: 
```
tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found 
```

I already installed cudnn with conda. What can I do to get it working?

**Comment by Felixkruemel** [comment, marked as answer]
Okay I fixed it.

Needed cudatoolkit from conda too :)
```
conda install -c conda-forge cudatoolkit 
conda install -c conda-forge cudnn 
```


Now the question is which prediction model is the best one and whether the default lstm settings are okay or an increase in epochs or so would result in a better result

**Reply by DidierRLopes** [reply]
I've wrote some comments on this in here:
* https://github.com/DidierRLopes/GamestonkTerminal/issues/58
* https://github.com/DidierRLopes/GamestonkTerminal/issues/154

TL;DRL: LSTM should be the best, but sometimes simpler models are more reliable. Needs to be tuned, and backtesting is a new feature that has been implemented recently. Just set the end date for the past like this "-e yyyy-mm-dd" so that you can see how it would perform on past data.
